apps/framework-cli/src/infrastructure/console.rs

```diff
@@ -49,6 +49,9 @@ pub async fn post_current_state_to_console(
         .map(|fo| fo.data_model.clone())
         .collect();
 
+    olap::clickhouse::check_ready(configured_db_client)
```

Comment:
Occasionally `fetch_all_tables` fail on start-up because clickhouse is not yet ready.